dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -191,7 +191,7 @@ metrics-jmx/4.1.1//metrics-jmx-4.1.1.jar
 metrics-json/4.1.1//metrics-json-4.1.1.jar
 metrics-jvm/4.1.1//metrics-jvm-4.1.1.jar
 minlog/1.3.0//minlog-1.3.0.jar
-netty-all/4.1.51.Final//netty-all-4.1.51.Final.jar
+netty-all/4.1.51.Final//netty-all-4.1.63.Final.jar
```

Comment:
Oh, yeah, and seems like we should do `./dev/test-dependencies.sh --replace-manifest`:

```diff
-netty-all/4.1.51.Final//netty-all-4.1.63.Final.jar
+netty-all/4.1.63.Final//netty-all-4.1.63.Final.jar
```

Reply:
`./dev/test-dependencies.sh --replace-manifest:` 
thx @HyukjinKwon ~ learned this new skill ~